src/formats/notion/convert-to-md.ts

```diff
@@ -217,13 +234,40 @@ function fixEquations(body: HTMLElement) {
 	}
 }
 
+function stripToSentence(paragraph: string) {
+	const firstSentence = paragraph.match(/^[^\.\?\!\n]*[\.\?\!]?/)?.[0]
+	return firstSentence ?? ''
+}
+
+function isCallout(element: Element) {
```

Comment:
Can this function be simplified to something like

```ts
return !!(/callout|bookmark/.test(element.getAttribute('class'))
```

I'm not seeing what the nullish is adding since both nullish values and an empty string are going to going to skip that clause.

Reply:
getAttribute('class') might return undefined, which breaks the regex.